### PR TITLE
Complete Homework from Class 09

### PR DESCRIPTION
**Pull request by blackeaglejs**
comfort level: 4

### CODE REVIEW

**Comment by blackeaglejs** [comment]
@mobjohnson Please see above.

File: class09/zoheb/manual_app/app/views/manual/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>The Manual Enrollment App!</h1><br><br>
+
+<a href='/about'>About</a><br><br>
+<a href='enroll'>Enroll Someone</a><br><br>
```
Comment (mobjohnson) [inline class09/zoheb/manual_app/app/views/manual/index.html.erb]: Very good!

**Comment by mobjohnson** [comment]
Congrats on creating your first rails app of your own!!!

I cannot accept this pull request as it includes a file outside of your student folder. I have reviewed your homework and show you as completed for this homework assignment. You do not need to resubmit.

**Comment by blackeaglejs** [comment]
Thank you Martin!

Does it indicate which file that is? I will try and get that fixed so that
when I submit the pull request for my next homework it can be accepted.
ᐧ

Sincerely,
Zoheb Nensey

**Pull request closed without merging**